PENDING.md

```diff
@@ -3,6 +3,7 @@
 BREAKING CHANGES
 
 * Gaia REST API (`gaiacli advanced rest-server`)
+  * [gaia-lite] [\#2191](https://github.com/cosmos/cosmos-sdk/issues/2191) Split `POST /stake/delegators/{delegatorAddr}/delegations` into `PUT /stake/delegators/{delegatorAddr}/delegations`, `POST /stake/delegators/{delegatorAddr}/unbonding_delegations` and `POST /stake/delegators/{delegatorAddr}/redelegations`
```

Comment:
Why is one `PUT` and the other two `POST`?

Reply:
updating delegations to `POST` based on our discussion. Technically unbonds and redelegations are `PUT` now (since you're overwriting them), but will leave them as `POST` since we'll have the indexing on #2929